platform/spire/src/command.py

```diff
@@ -68,6 +71,9 @@ def configure(self, command: list, parser: argparse.ArgumentParser):
         add_dry_run_argument(parser, 'dry_run_outer')
         add_show_commands_argument(parser, 'show_commands_outer')
 
+    def short_doc(self):
+        return self.__doc__
+
```

Comment:
Why do we have to override the superclass's implementation? Isn't this just the same implementation as Mux?

Reply:
oops